.eslintrc.js

```diff
@@ -21,6 +21,7 @@ module.exports = {
   rules: {
     'comma-dangle': 'off',
     'eol-last': 'off',
+    'func-call-spacing': 'off',
```

Comment:
These two rules were conflicting with some TypeScript function types and the formatter.